catalog-rest-service/src/main/java/org/openmetadata/catalog/util/EntityUtil.java

```diff
@@ -347,14 +349,17 @@ public static void removeTagsByPrefix(TagDAO tagDAO, String fullyQualifiedName)
     tagDAO.deleteTagsByPrefix(fullyQualifiedName);
   }
 
-  public static List<TagLabel> mergeTags(List<TagLabel> list1, List<TagLabel> list2) {
-    List<TagLabel> mergedTags =
-        Stream.concat(
-                Optional.ofNullable(list1).orElse(Collections.emptyList()).stream(),
-                Optional.ofNullable(list2).orElse(Collections.emptyList()).stream())
-            .distinct()
-            .collect(Collectors.toList());
```

Comment:
I'm probably missing something obvious here
The stream concat + distinct seems correct to me